.github/workflows/renovate.yaml

```diff
@@ -0,0 +1,35 @@
+name: Renovate
```

Comment:
I think the answer is no but are we ever likely to re-use this workflow? If so it might be better having this call a shared workflow

Reply:
Very very unlikely.

Reply:
No, very very unlikely.